### DISCUSSION TITLE
Translate

### DISCUSSION THREAD

**Discussion by datdv260497** [OrchardCMS/OrchardCore#10091] [Q&A]
Can I translate words that I add to the admin menu ?
Thank you !
![image](https://user-images.githubusercontent.com/49801948/128969266-6faa5af4-6484-480f-865e-ccfac83e9d9a.png)

**Comment by hishamco** [comment]
Dynamic translation is still in progress, for more information please refer to #4466

**Reply by Skrypt** [reply]
The alternative is to use a custom admin theme and use your own razor templates which make use of T["something to translate"]
We won't make these changes in source code because it's not the proper way to do this but in the mean time nothing prevents you to override these views in a custom theme or module.

**Comment by hishamco** [comment]
Dynamic translation is still in progress, for more information please refer to #4466
